book/standard_library.md

```diff
@@ -14,32 +18,218 @@ The standard library currently includes:
 To see a complete list of the commands available in the standard library, run the following:
 
 ```nu
-use std
-scope commands
-| where name =~ '^std '
-| select name usage extra_usage
-| wrap "std-lib"
+nu -c "
+  use std
+  scope commands
+  | where name =~ '^std '
+  | select name description extra_description
+  | wrap 'Standard Library Commands'
+  | table -e
+"
+```
+
+::: note
+The `use std` command above loads the entire standard library so that you can see all of the commands at once. This is typically not how it will be used (more info below). It is also run in a separate Nu subshell simply so that it is not loaded into scope in the shell you are using.
+:::
+
+## Importing the Standard Library
+
+While working at the commandline, it can be convenient to load the entire standard library using:
+
+```nu
+use std *
+```
+
+However, this form should be avoided in custom commands and scripts since it has the longest load time. While the load time is only on the order of ~20-30 milliseconds on an average system, that can add up in certain scenarios.
```

Comment:
What is an average system? Also, the time might change if we modify add more stuff to the stdlib.

Reply:
Fair enough - removed the second sentence.